app/screens/user_profile/user_profile.js

```diff
@@ -186,6 +190,46 @@ export default class UserProfile extends PureComponent {
         return null;
     };
 
+    buildCustomStatusBlock = () => {
+        const {formatMessage} = this.context.intl;
+        const {customStatus, theme, isMyUser} = this.props;
+        const style = createStyleSheet(theme);
+        const isStatusSet = customStatus.emoji || customStatus.text;
```

Comment:
this is repeated quite often. Maybe you could add that to the customStatus object itself? `customStatus.isStatusSet`

Reply:
probably a helper function would be better as you don't need to create an object with `isStatusSet` if the feature is not even enabled.